### PR TITLE
Adding sequence number functionality, closes #5

### LINKED ISSUE #5: [feature request] Independent Counter
I think currently the counter is incremented if the rest of the filename is the same.

Would it possible to add an option that increments the counter independently from the rest of the filename?

For example, currently:
With the following files existing in the folder
```
000001_good_image.png
000003_bad_image.png
```
and generating another `good_image` the filename is: `000002_good_image.png`

My request would make it possible for the filename to be `000004_good_image.png` instead.

Or in other words, what I am looking for, is for each file in the folder to have its unique ascending number.

### PR DESCRIPTION
**Pull request by AlUlkesh**
This adds a per-directory sequence number, as discussed in #5.

### CODE REVIEW

**Review by thedyze — CHANGES_REQUESTED** [review]

File: save_image_extended.py [hunk 1]
```diff
@@ -310,9 +325,17 @@ def save_images(self,
 							metadata.add_text(x, json.dumps(extra_pnginfo[x]))
 
 				if counter_position == 'last':
-					file = f'{filename}{delimiter_char}{counter:0{counter_digits}}.png'
```
[the remark this answers is not in the document]
Comment (AlUlkesh) [reply save_image_extended.py:313]: Not removed, changed to a version without .png and added at the end, so we don't up with .png.png. However probably a moot point due to the next remark.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,9 @@ git clone https://github.com/thedyze/save-image-extended-comfyui
 - `save_metadata` - Saves metadata into the image.
 - `counter_digits` - Number of digits used for the image counter. `3` = image_001.png. Will adjust the counter if files are deleted. Looks for the highest number in the folder, does not fill gaps.
 - `counter_position` - Image counter first or last in the filename.
+- `sequence_num` - Adds a per-directory sequence number to the filename.
+- `sequence_num_digits` - Number of digits used for the sequence number.
```
[the remark this answers is not in the document]
Comment (AlUlkesh) [reply README.md:30]: That's a good idea with a toggle instead. I'lll get on it.

File: assets/example.png
[the remark this answers is not in the document]
Comment (AlUlkesh) [reply assets/example.png:1]: Not by design. The top part is a screenshot from how it looks for me and I pasted your image only to find out you somehow have a wider area. I'll see what I can do.

[the remark this answers is not in the document]
Comment (thedyze) [reply assets/example.png:1]: The wider area is simply me stretching the node :)

File: save_image_extended.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (thedyze) [reply save_image_extended.py:313]: Ah, I see now, missed the part at the end before.

**Comment by AlUlkesh** [comment]
Here's the change to toggle. One more advantage is that this needs a lot less change of the original code:

https://github.com/thedyze/save-image-extended-comfyui/compare/d712114afa75dd61c9dde6c9cb896e8590212f6b...85e2de0dcec44b375fa816da0bf5e644e8891034#diff-a643104020a7483753628b7986091333cf05d458e463cb6b250fa5e6f3acfe42

**Comment by aegis72** [comment]
Not part of the issue, but just a comment: It's great to see an awesome, responsive maintainer!

**Review by thedyze — APPROVED** [review]
Looks good!

**Pull request merged**